projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -19,9 +19,9 @@
             formatter
                 ? (value | columnFormatter: formatter)
                 : column.dataType === 'number'
-                ? (value | igxdecimal: grid.locale)
+                ? (value | number: undefined:grid.locale)
```

Comment:
Don't pass `undefined` as format argument, just expose a new input for this in the grid column.

Reply:
@kdinev lets discuss that on the sos meeting, we had some concerns for exposing input/s for the default cell template as it would cost us setting up to 3 inputs for each column (considering scenario with a grid a lot of columns).

And with that said, we considered that if the end-user wants to set a custom format/timeZone/locale, defining an ng-template with the desired pipe config is always an option. After all we strive to provide a default handling for value of type date, number and etc. which implies that we define what 'format', 'timeZone' and 'locale' should be set by default, and if the end-user wants to change that, a cell template should be used (as they are used to do).

Reply:
After discussion, it was decided that `format` input and `timeZone` inputs will be exposed. This way, a dev can give a specific format for each date/numeric column. Locale used will be grid.locale. Examples:

     * <igx-column dataType="date" timeZone="UTC" format="longDate"></igx-column>
     * <igx-column dataType="number" format="4.5-5"></igx-column>

As seen, `format` input is used for both date and numeric columns.

Reply:
We had a follow-up meeting with @rkaraivanov and @kdinev that resulted in the following - create IColumnPipeArgument interface with the following optional properties:

```ts
IColumnPipeArguments: {​​​​
    digitsInfo: string
    format: string;
    timeZone: string
}​​​​
```

Reply:
@kdinev what about the naming for the actual column property - we talked about `formatOptions`?